packages/theme/src/theme/_buttons.scss

```diff
@@ -76,6 +76,43 @@
 		padding: 0 $padding-smaller;
 	}
 
+	&-active-icon {
+		background-color: unset;
+		border: 1px solid #333;
+		border-radius: 12px;
+		box-shadow: none;
+		color: #333;
+		height: 24px;
+		line-height: unset;
+		opacity: 0.5;
+		padding: 0;
+		width: 24px;
```

Comment:
The width/height/radius are magicnumber. Plus, the button size seems very small compared to the buttons.
Can't we align the sizes with btn's ones ?